scripts/install_xcpng.py

```diff
@@ -174,6 +181,11 @@ def main():
              "In case of a restore, specify the version of the installer."
     )
     parser.add_argument("--installer", help="URL of the installer")
+    parser.add_argument(
+        "--vm_hostname",
```

Comment:
Underscores in parameter names are not really common.  This would be `--vm-hostname` , but at some point in the future we might extend the script to handle physical hosts, and so `vm` would become outdated. Maybe `--set-hostname`, or just `--hostname` (I know there's a little risk of confusion with the host parameter, but with the help text I think it's good enough)

Reply:
Yes I was concerned by the confusion between `hostname` and `host` (hostname_or_ip).
What about `--xcp-ng-hostname`  ?

Reply:
It's equally confusing, because the other hostname is also a XCP-ng hostname :D. I thought of `--target-hostname` but it may be a bit long.

Reply:
Let's try target hostname. Too long is better than too confusing :)

Reply:
Works for me. And with argparse you can easily add a `-h` short version of the parameter.

Reply:
Or  `-t` if `-h` conflicts with the help :grimacing: